indico/modules/rb_new/client/js/common/timeline/MonthlyTimelineContent.jsx

```diff
@@ -73,38 +81,47 @@ export default class MonthlyTimelineContent extends WeeklyTimelineContent {
     renderDividers() {
         const nDays = this.dates.length;
         const daySize = (100 / nDays);
-
         const {dateRange} = this.props;
         const emptyDays = this.dates
             .filter(day => dateRange.length !== 0 && !dateRange.includes(day))
             .map(day => this.dates.findIndex((el) => el === day));
 
         return (
-            _.times(nDays, n => (
-                <div styleName="style.timeline-day-divider"
-                     className={toClasses({hidden: emptyDays.includes(n), visible: !emptyDays.includes(n)})}
-                     style={{left: `${n * daySize}%`, width: `${daySize}%`}}
-                     key={`day-divider-${n}`} />
-            ))
+            _.times(nDays, n => {
+                const indicateWeekend = this.weekendDays.includes(n) && !emptyDays.includes(n);
+                const dividerStyle = 'style.timeline-day-divider';
+                return (
+                    <div styleName={indicateWeekend ? `${dividerStyle} style.weekend` : dividerStyle}
+                         className={toClasses({hidden: emptyDays.includes(n), visible: !emptyDays.includes(n)})}
+                         style={{left: `${n * daySize}%`, width: `${daySize}%`}}
+                         key={`day-divider-${n}`} />
+                );
+            })
         );
     }
 
     renderHeader() {
         const {longLabel, selectable} = this.props;
         const labelWidth = longLabel ? 200 : 150;
+
         return (
             <>
-                <div styleName="baseStyle.timeline-header" className={!selectable ? 'timeline-non-selectable' : ''}>
+                <div styleName="baseStyle.timeline-header"
+                     className={!selectable ? 'timeline-non-selectable' : ''}>
                     <div style={{minWidth: labelWidth}} />
                     <div styleName="style.timeline-header-labels">
-                        {_.map(this.dates, (dt, n) => (
-                            <div styleName="style.timeline-header-label"
-                                 key={`timeline-header-${n}`}>
-                                <span styleName="style.timeline-label-text">
-                                    {toMoment(dt, 'YYYY-MM-DD').format('D')}
-                                </span>
-                            </div>
-                        ))}
+                        {_.map(this.dates, (dt, n) => {
```

Comment:
I am curious why we use `_.map` when we could use `Array.prototype.map` instead.